Makefile

```
@@ -106,6 +106,25 @@ verify: tidy fmt vet generate ## Verify the current code generation and lint
 lint: $(GOLANGCI_LINT) ## Run golangci linter.
 	$(GOLANGCI_LINT) run $(GOLANGCI_LINT_ARGS)
 
+.PHONY: test-upgrade-e2e
+test-upgrade-e2e: export TEST_CLUSTER_CATALOG_NAME := test-catalog
+test-upgrade-e2e: kind-cluster cert-manager build-container kind-load image-registry run-latest-release wait pre-upgrade-setup deploy  ## Run upgrade e2e tests on a local kind cluster
+test-upgrade-e2e:
+	${MAKE} wait
+	${MAKE} post-upgrade-checks
+	${MAKE} kind-cluster-cleanup
```

Wondering if we can just do something like:

```suggestion
test-upgrade-e2e: kind-cluster cert-manager build-container kind-load image-registry run-latest-release pre-upgrade-setup deploy post-upgrade-checks kind-cluster-cleanup  ## Run upgrade e2e tests on a local kind cluster
```